lib/index.js

```diff
@@ -83,22 +85,39 @@ function normalizePaths (value) {
   }
 
   const cwd = process.cwd()
-  const homeDir = os.homedir()
+  const cwdReal = getRealPath(cwd)
   const tempDir = os.tmpdir()
+  const tempDirReal = getRealPath(tempDir)
+  const homeDir = os.homedir()
+  const homeDirReal = getRealPath(homeDir)
 
   const homeRelativeToTemp = path.relative(tempDir, homeDir)
+  const homeRelativeToTempReal = path.relative(tempDirReal, homeDir)
+  const homeRealRelativeToTempReal = path.relative(tempDirReal, homeDirReal)
+  const homeRealRelativeToTemp = path.relative(tempDir, homeDirReal)
 
   const runner = [
     // Replace process.cwd with <PROJECT_ROOT>
+    val => val.split(cwdReal).join('<PROJECT_ROOT>'),
     val => val.split(cwd).join('<PROJECT_ROOT>'),
+
     // Replace home directory with <TEMP_DIR>
+    val => val.split(tempDirReal).join('<TEMP_DIR>'),
     val => val.split(tempDir).join('<TEMP_DIR>'),
+
     // Replace home directory with <HOME_DIR>
+    val => val.split(homeDirReal).join('<HOME_DIR>'),
     val => val.split(homeDir).join('<HOME_DIR>'),
+
     // handle HOME_DIR nested inside TEMP_DIR
     val => val.split(`<TEMP_DIR>${path.sep + homeRelativeToTemp}`).join('<HOME_DIR>'),
+    val => val.split(`<TEMP_DIR>${path.sep + homeRelativeToTempReal}`).join('<HOME_DIR>'), // untested
```

Comment:
If you have an idea how to test these two "untested" lines, please add to this PR or give me an idea how I should. I personally think it is okay to merge without these tested as they are safeguards.

Reply:
Yep, I just ran this patch against a couple projects and all existing functionality is still running fine. Also, you've increased our test suite 10x 🙌 so I don't have a problem merging. If anyone actually encounters this edge-case, we can open a new issue.